### PR TITLE
Add samples guide in the help panel

### PR DESCRIPTION
**Pull request by NipuniBhagya**

### CODE REVIEW

File: apps/developer-portal/src/constants/documentation-constants.ts [hunk 1]
```diff
@@ -27,13 +27,19 @@ export class DocumentationConstants {
...
 
     /**
      * Key to find the developer portal docs.
      * @constant
      * @type {string}
      */
     public static readonly PORTAL_DOCS_KEY = "[\"Developer Portal\"]";
+
+    public static readonly Samples_Catalog: Map<string, string> = new Map<string, string>()
+        .set("JS_SPA_SAMPLE", "https://github.com/wso2-extensions/identity-samples-js/releases/download/0.1.0/" +
```
Comment (jeradrutnam) [inline apps/developer-portal/src/constants/documentation-constants.ts:43]: We'll get this to a config. Do it in another PR

File: apps/developer-portal/src/components/applications/help-panel/samples-guide.tsx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        } else if (protocolName === "wstrust") {
+            protocolName = SupportedAuthProtocolTypes.WS_TRUST;
+        } else if (protocolName === "samlsso") {
+            protocolName = SupportedAuthProtocolTypes.SAML;
+        }
+
+        return protocolName;
+    });
+
+    const downloadConfigFile = () => {
```
Comment (DimalChandrasiri) [inline apps/developer-portal/src/components/applications/help-panel/samples-guide.tsx:148]: Add return as void if nothing is returned.

Comment (NipuniBhagya) [reply apps/developer-portal/src/components/applications/help-panel/samples-guide.tsx]: Resolved with fb5681d
